### PR TITLE
[MODULAR] Armory Spawn Shotguns Fix

### PR DESCRIPTION
**Pull request by MassacreDemigirl**
## About The Pull Request
Makes the shotguns that spawn in armory which are Model 23-37 and M2 Auto-Shotgun back into 12 Gauge as well as fixes the descriptions of those two shotguns and SAS14

## How This Contributes To The Skyrat Roleplay Experience
14 Gauge was meant to be used with SAS14 due to being a magazine fed shotgun and comes with its own perks and quirks. However this means security has no proper viable way of resupplying those two shotguns and also 14 Gauge damage model means using those two shotguns are worthless when combined with the fact the player has to handload each shell unlike SAS14.

## Changelog

:cl:
fix: Model 23-37 and M2 Auto are now 12 gauge again. Also fixes the descriptions of the Armory Shotguns
/:cl:

### CODE REVIEW

**Review by tf-4 — APPROVED** [review]
Debating on calling this a balance change but tbh it's a dumb thing that's been like this for so long I'm just glad someone bothered to change it

File: modular_skyrat/modules/sec_haul/code/guns/guns.dm [hunk 1]
```diff
@@ -693,7 +693,7 @@
 
 /obj/item/gun/ballistic/shotgun/m23
 	name = "\improper Model 23-37"
-	desc = "A common outdated police shotgun sporting an eight-round tube."
+	desc = "A common outdated police shotgun sporting an six-round tube."
```
Comment (nevimer) [inline modular_skyrat/modules/sec_haul/code/guns/guns.dm]: a six round tube

**Comment by tf-4** [comment]
Also committing from master = based

**Comment by MassacreDemigirl** [comment]
I also edit the code from web editor

(Yes i playtested it)

**Pull request merged**